internal/statistics/container_statistics.go

```diff
@@ -46,32 +50,68 @@ func (cs containerStatistic) logger() zerolog.Logger {
 		Logger()
 }
 
+func (cs containerStatistic) appendInitFields(event *zerolog.Event) {
+	if !cs.runningTimestamp.IsZero() && cs.previous != nil && !cs.previous.readyTimestamp.IsZero() {
```

Comment:
`cs.runningTimeStamp` is always != nil ?

Reply:
yeah, it's a `time.Time` not a pointer to `time.Time` so it's the `time.Time` zero value until set, which is—I believe—the unix epoch.

Reply:
So yes technically this code won't work before 1970

Reply:
(it was a joke, happy to see that someone read my comment 😀 )